client/container_start.go

```diff
@@ -9,7 +9,9 @@ import (
 // ContainerStart sends a request to the docker daemon to start a container.
 func (cli *Client) ContainerStart(ctx context.Context, containerID string, checkpointID string) error {
 	query := url.Values{}
-	query.Set("checkpoint", checkpointID)
+	if len(checkpointID) != 0 {
```

Comment:
Hum if `checkpointID` is optional (which seems it is), then I would definitely move it to a `Option` struct, even for one fiead… @stevvooe @calavera WDYT ?